src/game_inventory.cpp

```diff
@@ -25,20 +57,13 @@ void game::interactive_inv()
     } while( allowed_selections.count( res ) != 0 );
 }
 
-item_location_filter convert_filter( const item_filter &filter )
-{
-    return [ &filter ]( const item_location & loc ) {
-        return filter( *loc );
-    };
-}
-
-int game::inv_for_filter( const std::string &title, item_filter filter,
+int game::inv_for_filter( const std::string &title, const item_filter &filter,
```

Comment:
Presumably this is an interface for legacy code? Now `item_location` can be serialized to JSON we should deprecate referring to items by inventory position. The function declaration could be marked with a warning as such.